src/plugins/discover/public/application/main/components/sidebar/discover_field.tsx

```diff
@@ -397,6 +397,9 @@ function DiscoverFieldComponent({
 
   const renderPopover = () => {
     const details = getDetails(field);
+
+    // TODO: integrate <FieldStats .../>
```

Comment:
I am going to reintroduce [these changes to Discover](https://github.com/elastic/kibana/pull/136328/commits/3f4ae6e3959e90528ac2be28d00d71904cf9eb25) in a follow up PR.